pkg/skaffold/kubernetes/portforward/forwarder_manager.go

```diff
@@ -38,20 +43,69 @@ type ForwarderManager struct {
 }
 
 // NewForwarderManager returns a new port manager which handles starting and stopping port forwarding
-func NewForwarderManager(out io.Writer, cli *kubectl.CLI, podSelector kubernetes.PodSelector, namespaces []string, label string, opts config.PortForwardOptions, userDefined []*latest.PortForwardResource) *ForwarderManager {
+func NewForwarderManager(out io.Writer, cli *kubectl.CLI, podSelector kubernetes.PodSelector, namespaces []string, label string, runMode config.RunMode, options config.PortForwardOptions, userDefined []*latest.PortForwardResource) *ForwarderManager {
+	if !options.Enabled() {
+		return nil
+	}
+
+	// TODO this doesn't feel like the right place
+	if err := options.Validate(); err != nil {
```

Comment:
Shd we define a custom flag type in `cmd/skaffodl/app/flag` in validate in `Set` method.

e.g. `BuildOutputFileFlag`

Reply:
`PortForwardOptions` is now a `SliceValue`.